src/main/java/de/yggdrasil/core/dal/data/datasources/RabbitMQ.java

```diff
@@ -24,11 +28,21 @@ public class RabbitMQ implements EventDataSource<RabbitMQMessage, DALDefaultEven
     private Channel channel;
     private final DALEventbus eventbus = new DALDefaultEventBus();
 
+    /**
+     * The callback variable is a private final variable of type DeliverCallback.
+     * It is used to handle incoming messages from RabbitMQ.
```

Comment:
```suggestion
     * used to handle incoming messages from RabbitMQ.
```
The description of the field modifiers aswell as the description of the type is unnecessary.